MIGRATING.md

```diff
@@ -353,7 +361,7 @@ export const handlers = [
 When you provide an object as the `ResponseInit.headers` value, you cannot specify multiple response cookies with the same name. Instead, to support multiple response cookies, provide a `Headers` instance:
 
 ```js
-import { Headers, HttpResponse, rest } from 'msw'
+import { Headers, http, HttpResponse } from 'msw'
```

Comment:
While we're at it, maybe let's clean up other global classes like `Headers` too? No need to import them from MSW anymore, neither does it export them with 2.0.